src/modm/driver/encoder/bitbang_encoder_output_impl.hpp

```diff
@@ -10,79 +10,72 @@
  */
 // ----------------------------------------------------------------------------
 
-#ifndef	MODM_ENCODER_OUTPUT_HPP
-#	error	"Don't include this file directly, use 'encoder_output.hpp' instead!"
+#ifndef MODM_BITBANG_ENCODER_OUTPUT_IMPL_HPP
```

Comment:
```suggestion
#ifndef MODM_BITBANG_ENCODER_OUTPUT_HPP
```

Reply:
oops... did not check building the encoder_output example.
Next time i'll do 😅

Reply:
😫 Just discovered this **Commit suggestion** button below reviews. It could have been so easy, haha.